### DISCUSSION TITLE
Enable Localization

### DISCUSSION THREAD

**Discussion by Gabor111a** [lukevella/rallly#150] [Ideas]
A solution for localization would be welcome.
I could already contribute to translate to French and Hungarian.

**Comment by lukevella** [comment]
I need to research ways that will allow me to gather user submitted translations. Something like [weblate](https://weblate.org/en-gb/) or [transifex](https://www.transifex.com/). If anyone has any suggestions about what's best let me know.

**Reply by Gabor111a** [reply]
[weblate](https://weblate.org/en-gb/) has self-hosted gratis plan and 'Gratis Weblate' for libre projects (see bottom of [features](https://weblate.org/en-gb/features/) page),
[transifex](https://www.transifex.com/) seems having only paid plans.
There is also [Loco](https://localise.biz) which has free plan (2000 strings limit).

**Reply by Eirikr70** [reply]
I might also help with french or italian translation.

**Reply by lukevella** [reply]
Hi, if you're still interested with helping I wrote a guide on how to get started: https://github.com/lukevella/rallly/wiki/Guide-for-translators

**Reply by joho1968** [reply]
> Hi, if you're still interested with helping I wrote a guide on how to get started: https://github.com/lukevella/rallly/wiki/Guide-for-translators

I've made some corrections to the Swedish translation. In particular the translation of "poll", which normally would be "enkät", but in this context is better as "förfrågan", if you take into consideration that Rallly is more about a "meeting request with options" rather than a "poll".

**Reply by lukevella** [reply]
Hi @joho1968, I'll have to take your word for it but if you can open a discussion on Crowdin about such changes that would be great so that other translators can get involved.

Here's a link: https://crowdin.com/project/rallly/discussions

**Reply by joho1968** [reply]
> Hi @joho1968, I'll have to take your word for it but if you can open a discussion on Crowdin about such changes that would be great so that other translators can get involved.

Done: https://crowdin.com/project/rallly/discussions/5
